pkg/metrics/metrics.go

```diff
@@ -28,17 +28,19 @@ var (
 
 // gauges
 var (
-	// DEPRECATED - See UserAccountGaugeVec
+	// DEPRECATED - See UserAccountGaugeVec and MasterUserRecordGaugeVec
 	// MasterUserRecordGauge should reflect the current number of master user records in the system
 	MasterUserRecordGauge prometheus.Gauge
 )
 
 // gauge with labels
 var (
-	// UserAccountGaugeVec should reflect the current number of master user records in the system, with a label to partition per member cluster
+	// UserAccountGaugeVec reflects the current number of master user records in the system, with a label to partition per member cluster
 	UserAccountGaugeVec *prometheus.GaugeVec
-
+	// UsersPerActivationGaugeVec reflects the number of users labelled with on their current number of activations
 	UsersPerActivationGaugeVec *prometheus.GaugeVec
+	// MasterUserRecordGaugeVec reflects the current number of MasterUserRecords, labelled with their email address domain (`@redhat.com` vs `@*.ibm.com` vs others)
```

Comment:
```suggestion
	// MasterUserRecordGaugeVec reflects the current number of MasterUserRecords, labelled with their email address domain (`internal` vs `external`)
```
I guess this should now refer to internal and external?